apps/tasks/lib/controllers/assignee_select_controller.dart

```diff
@@ -41,13 +42,15 @@ class AssigneeSelectController extends ChangeNotifier {
     load();
   }
 
+  final Logger _logger = Logger();
+
   /// Loads the tasks
   Future<void> load() async {
     state = LoadingState.loading;
     String? currentOrganization = CurrentWardService().currentWard?.organizationId;
     if(currentOrganization == null){
       if(kDebugMode){
-        print("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
+        _logger.w("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
```

Comment:
Unsure whether below is a better alternative, here at least we don't have to declare a logger on every Object, or maybe we can use a logger service
```suggestion
        Logger().w("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
```

Reply:
> Unsure whether below is a better alternative, here at least we don't have to declare a logger on every Object, or maybe we can use a logger service

I think a logger service would be the best